contents/blog/customer-support-at-posthog.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-05-17
+title: How We Do Customer Support At Our Open-Source DevTool Company
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [Mohammed Shehu](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+Customer support is a critical function of any startup, especially if you offer a highly technical product like we do. [Charles Cook](https://www.linkedin.com/in/charlescook1/), Business Head at PostHog, shared his thoughts on how CS was carried out at his previous companies (VITL and Roli) and how customer support has evolved here at PostHog over the past few months. 
+
+[Roli](http://www.roli.com/) sells musical instruments and production software. Because the company deals with creators, the customer support team consists of qualified technical musicians, producers, and composers in their own right. Reflecting the makeup of your customer base is key to ensuring your CS team can relate to the problems that your customers face and provide better, more effective solutions.
+
+[VITL](https://vitl.com/) was a similar story: the company provides personalized nutrition, tailor-made supplements, and at-home nutrition tests to its customers. During Charles' time there, the team was more hybrid, with some of the CS staffers spending 100% of their time on customer support and others splitting their time between support and working on nutrition. Again, this meant that the support team consisted of qualified professionals who could nail down problems and serve up credible solutions.
```

Comment:
Don't use acronyms ie "CS" without clarifying it earlier on - lots of developers would read CS as Computer Science